cmd/operator-sdk/alpha/cmd.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/alpha/olm"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "alpha",
+		Short: "Run an alpha subcommand",
```

Comment:
Should we list which subcommands are under alpha?

Reply:
The cobra help does this for us automatically:

```console
$ operator-sdk alpha -h
Run an alpha subcommand

Usage:
  operator-sdk alpha [command]

Available Commands:
  olm         Manage the Operator Lifecycle Manager installation in your cluster

Flags:
  -h, --help   help for alpha

Global Flags:
      --verbose   Enable verbose logging

Use "operator-sdk alpha [command] --help" for more information about a command.
```